pychunkedgraph/app/segmentation/common.py

```diff
@@ -586,14 +587,20 @@ def handle_leaves(table_id, root_id):
     else:
         bounding_box = None
 
-    # Call ChunkedGraph
     cg = app_utils.get_cg(table_id)
-    atomic_ids = cg.get_subgraph(
-        int(root_id), bbox=bounding_box, bbox_is_coordinate=True, leaves_only=True
+    if stop_layer > 1:
+        return cg.get_subgraph_nodes(
```

Comment:
this function _should_ return a dictionary but we discussed to return a flattened view of the values. Either way it is, something is missing

Reply:
the endpoint is only for a single root ID though, it will return an array when a sinlge ID is passed
`/table/<table_id>/node/<node_id>/leaves`

Reply:
But if `stop_layer > 1` then this function should return a dictionary returned with `layer -> array[ids]` mapping. We discussed that we want to flatten the values. Where is the flattening happening though?

Reply:
I don't believe this would flatten right now? But I could be wrong.

Reply:
okay I misunderstood, I thought you were talking about dict of root IDs to children, will check

Reply:
What should be the response when a `stop_layer` is given? Should it return children at `<= stop_layer` or `>= stop_layer` ?
Also thinking of returning layers as a separate array, optionally with a param `return_layers=True` because the response needs to be flattened